compliance_checker/suite.py

```diff
@@ -83,10 +101,21 @@ def load_all_available_checkers(cls):
         Helper method to retrieve all sub checker classes derived from various
         base classes.
         """
-        for x in working_set.iter_entry_points('compliance_checker.suites'):
+        cls._load_checkers(working_set
+                           .iter_entry_points('compliance_checker.suites'))
+
+    @classmethod
+    def _load_checkers(cls, checkers):
+        """
+        Loads up checkers in an iterable into the class checkers dict
+        :param checkers: An iterable containing the checker objects
+        """
+
+        for c in checkers:
             try:
-                xl = x.resolve()
-                cls.checkers[':'.join((xl._cc_spec, xl._cc_spec_version))] = xl
+                check_obj = c.resolve()
+                cls.checkers[':'.join((check_obj._cc_spec,
+                                       check_obj._cc_spec_version))] = check_obj
             # TODO: remove this once all checkers move over to the new
             #       _cc_spec, _cc_spec_version
             except AttributeError:
```

Comment:
this is probably fine, but maybe a future cleanup to explicitly check for `_cc_spec`/`_cc_spec_version` with `hasattr` instead of assuming the `AttributeError` has to do with these two lookups.